app/assets/javascripts/application.js

```diff
@@ -1,18 +1,5 @@
-function ready (fn) {
-  if (document.readyState !== 'loading') {
-    // IE9 support
-    fn()
-  } else {
-    // Everything else
-    document.addEventListener('DOMContentLoaded', fn)
-  }
-}
-
-// Warn about using the kit in production
-if (window.console && window.console.info) {
-  window.console.info('GOV.UK Prototype Kit - do not use for production')
-}
+// Add extra JavaScript here
```

Comment:
I think its simplest and safest to just have the prompt on line 4 maybe

Reply:
Happy for the content of this placeholder file to be rewritten.

Reply:
A simple cypress test to inject some javascript into the application.js and then test that it runs should be quite simple and that would cover testing that part of this ticket.

Reply:
@BenSurgisonGDS good idea, I'll do that.

Reply:
This was added in #1488.